### PR TITLE
Ana/fix avatars broken

### LINKED ISSUE #4092: Avatar broken
Avatar was updated in #4089 and it seems it broke it:

I take the opportunity to reiterate my proposal to only update packages weekly, so we can focus on monitor breaking changes the "updating" day of the week.

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4092

**Description:**

Fixed avatars after update #4089 . Unfortunately I am too lazy to open an issue on official repo https://github.com/DiceBear/avatars.

They should update their documentation

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/Avatar.vue [hunk 1]
```diff
@@ -4,23 +4,23 @@
...
+    },
   },
   computed: {
     svg() {
       const options = {}
       const avatars = new Avatars(sprites, options)
-      return avatars.create(this.address)
-    }
-  }
+      return avatars.create(this.address, options)
```
Comment (faboweb) [inline src/components/common/Avatar.vue:22]: this is needed? https://github.com/DiceBear/avatars/tree/master/packages/avatars

Comment (Bitcoinera) [reply src/components/common/Avatar.vue:22]: that's why I should open an issue to these guys 😅 It is in the PR description. Documentation outdated at least for jidenticons

**Comment by faboweb** [comment]
Did you update your dependencies before this? I don't see a difference in the previews

https://deploy-preview-4093--lunieio.netlify.app/cosmos-hub-testnet/transactions
https://develop--lunieio.netlify.app/cosmos-hub-testnet/transactions

**Comment by Bitcoinera** [comment]
OK, let's pretend like this never happened 😅 will you?

**Comment by Bitcoinera** [comment]
MizAndMrs-television-miz-and-mrs-mizandmrs-QZnpeNW0lOaDj30jNP

File: src/components/common/Avatar.vue (hunk 1 above)
Comment (FlorianKoerner) [reply src/components/common/Avatar.vue:22]: Sure? Actually the old version should work too. If not, can you describe the problem? 😅

Comment (Bitcoinera) [reply src/components/common/Avatar.vue:22]: Hi Florian! Actually this was me messing things up 😅 Don't worry, your new version works great too with the current documentation.

I just needed to update my packages, it was as simple as that.

Sorry for the confusion here 🙏 
And thank you for being so attentive!

**Pull request closed without merging**